alpha/lib/model/entry.php

```diff
@@ -2794,7 +2794,39 @@ public function postUpdate(PropelPDO $con = null)
 				$redirectEntry->save();
 			}
 		}
-		
+
+		//Has entitledUserPuserEdit/Publish changed for current entry
+		if ( ($this->oldCustomDataValues['']['entitledUserPuserEdit'] != $this->getEntitledPusersEdit()) ||
```

Comment:
Please add tiny method getOldCustomDataValues(fieldName , nameSpace=null)
And validate that value exist.